conjureup/controllers/deploy/gui.py

```diff
@@ -335,14 +336,13 @@ def do_deploy_remaining(self):
 
     def finish(self):
         def enqueue_set_relations():
-            rel_future = juju.set_relations(self.applications,
-                                            app.ui.set_footer,
-                                            partial(self._handle_exception,
-                                                    "Error setting relations"))
+            rel_future = juju.set_relations(
+                self.applications,
+                app.ui.set_footer,
+                handle_exception)
```

Comment:
you still need partial here, as `handle_exception` requires an argument. might want to check for other places where you removed partial